db/meddler.go

```diff
@@ -57,6 +57,11 @@ func SlicePtrsToSlice(slice interface{}) interface{} {
 	return res.Interface()
 }
 
+// RegisterMeddlerType registers a new meddler type with the given name
+func RegisterMeddler(name string, meddlerType meddler.Meddler) {
+	meddler.Register(name, meddlerType)
+}
```

Comment:
Nitpick, why not directly using the `meddler.Register` function, instead of this wrapper function?

```suggestion
```

Reply:
Like I wrote in the comment where this is called, I need to ensure that `db` package and his `init` function is called before calling the register of a custom meddler.